src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,114 @@
+# archive_unstable_storageDiff
+
+**Parameters**:
+
+- `currentHash`: String containing a hexadecimal-encoded hash of the header of the block whose storage difference will be retrieved. The storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
```

Comment:
The second sentence is contradicting the `fromHash` parameter.

Reply:
Also I don't think that the naming is appropriate. What exactly is "current" here?

Reply:
Have removed the second sentance and provided the details in `previousHash` section, thanks!